presentation/src/main/java/com/anytypeio/anytype/presentation/sharing/AddToAnytypeViewModel.kt

```diff
@@ -71,9 +76,12 @@ class AddToAnytypeViewModel(
     val navigation = MutableSharedFlow<OpenObjectNavigation>()
     val spaceViews = MutableStateFlow<List<SpaceView>>(emptyList())
     val commands = MutableSharedFlow<Command>()
+    val progressState = MutableStateFlow<ProgressState>(ProgressState.Init)
 
     val state = MutableStateFlow<ViewState>(ViewState.Init)
 
+    private var createdWrapperObjId : String? = null
```

Comment:
Consider wrapping this into a sealed class with state: `Init, Uploading(process: Id, wrapperObjectId: Id)`

Reply:
done